jenkins/test_orchestrator/JenkinsFile

```diff
@@ -0,0 +1,62 @@
+pipeline {
+    agent any
+
+    parameters {
+        string(name: 's3_bucket'),
+        string(name: 'opensearch_version'),
+        string(name: 'build_id'),
+        string(name: 'architecture')
+    }
+
+    stages {
+        stage('Execute Tests') {
+            steps {
+                // To run all three test jobs in parallel, we use "parallel" step
+                // https://jenkins.io/doc/pipeline/examples/#jobs-in-parallel
+                parallel (
+                    IntegTests: {
+                        build job: 'integ-tests'
+                        parameters: [
+                            string(name: 's3_bucket', value: ${params.s3_bucket}),
```

Comment:
Artifact S3 bucket could be fetched via Jenkins env variable `ARTIFACT_BUCKET_NAME`